internal-services/catalog/iib-pipeline.yaml

```diff
@@ -18,12 +18,12 @@ spec:
       description: The Snapshot in JSON format
     - name: iibServiceConfigSecret
       type: string
-      description: Secret containing IIB service Config
```

Comment:
@theflockers I think this `description` was removed by mistake.

Reply:
It moved two lines down @happybhati .

Reply:
ah yes, I see.